src/compose/types/service.ts

```diff
@@ -186,7 +186,7 @@ export interface DeviceMetadata {
 	imageInfo?: Dockerode.ImageInspectInfo;
 	uuid: string | null;
 	appName: string;
-	version: string;
+	version: string; // TODO: this is not used anywhere
```

Comment:
What's the resistance behind removing this?

Reply:
I used VSCode get references for this, I'm not sure if there is a case that may not have been caught as part of a type reassignment. Searching the code for the string `"version"`, I'm pretty convinced it's not used. I'll remove it and hopefully we'll catch during testing if I missed something